packages/contracts-bedrock/src/libraries/DisputeTypes.sol

```diff
@@ -66,9 +66,8 @@ type GameType is uint8;
 type VMStatus is uint8;
 
 /// @notice The current status of the dispute game.
-enum GameStatus
-// The game is currently in progress, and has not been resolved.
-{
+enum GameStatus {
```

Comment:
Did this diff come from `forge fmt`?

Reply:
must have come from the linter, should I revert?

Reply:
You will likely need to, the `ci/circleci: contracts-bedrock-checks` will fail if there is a diff after linting